src/main/kotlin/com/hibob/project/resource/feedbackResource.kt

```diff
@@ -0,0 +1,44 @@
+package com.hibob.project.resource
+
+import com.hibob.project.dao.*
+import com.hibob.project.service.FeedbackService
+import jakarta.ws.rs.*
+import jakarta.ws.rs.container.ContainerRequestContext
+import jakarta.ws.rs.core.Context
+import jakarta.ws.rs.core.MediaType
+import jakarta.ws.rs.core.Response
+import org.springframework.stereotype.Controller
+
+@Controller
+@Path("/api/system/feedback")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+class FeedbackResource(private val feedbackService: FeedbackService) {
+    @GET
+    @Path("/{companyId}")
+    fun getAllFeedbacks(@PathParam("companyId") companyId: Long): Response {
+        feedbackService.getAllFeedbacks(companyId)
+
+        return Response.ok().build()
+    }
+
+    @POST
+    fun feedbackSubmission(@Context requestContext: ContainerRequestContext, feedbackCreationRequest: FeedbackCreationRequest): Response {
+        val authenticatedEmployee = authenticateEmployee(requestContext)
+        feedbackService.feedbackSubmission(feedbackCreationRequest, authenticatedEmployee)
+
+        return Response.ok(feedbackCreationRequest).build()
+    }
+
+    private fun authenticateEmployee(requestContext: ContainerRequestContext): LoginEmployeeResponse {
```

Comment:
maybe rename the function? it doesn't do authentication, the user is already authenticated.

Also- are you going to have other resource classes? if so, should this be here specifically?